pandas/tests/sparse/test_series.py

```diff
@@ -1361,3 +1364,89 @@ def test_numpy_func_call(self):
         for func in funcs:
             for series in ('bseries', 'zbseries'):
                 getattr(np, func)(getattr(self, series))
+
+
+def test_constructor_dict():
+    d = {'a': 0., 'b': 1., 'c': 2.}
```

Comment:
you might be able to move some of these into ``from pandas.tests.series.test_api import SharedWithSparse`` whech we already import (rather than directly copying them).